gw_spaceheat/template_settings.py

```diff
@@ -0,0 +1,3 @@
+#MQTT related
```

Comment:
I like this format of having the template settings and a gitignored `settings.py` 👍🏾